ndcube/ndcube.py

```diff
@@ -1208,13 +1208,21 @@ def my_propagate(uncertainty, data, mask, **kwargs):
         # Resample WCS
         new_wcs = ResampledLowLevelWCS(self.wcs.low_level_wcs, bin_shape[::-1])
 
+        # If meta is axis-aware, drop axis-awareness for metadata associated with rebinned axes.
+        if hasattr(self.meta, "__ndcube_can_rebin__") and self.meta.__ndcube_can_rebin__:
+            new_shape = (np.asarray(self.shape) / np.asarray(bin_shape)).astype(int)
+            rebinned_axes = set(np.where(bin_shape != 1)[0])
+            new_meta = self.meta.rebin(rebinned_axes, new_shape)
+        else:
+            new_meta = self.meta
```

Comment:
should this have always copied the meta? :thinking:

Reply:
Not sure what you mean here.

Reply:
I assume `self.meta.copy()`

Reply:
I mean should rebin have always made a copy of meta, i.e so the rebinned meta isn't the same object as the parent?

Reply:
Ah.  I guess yes.  I'll make this line include copy.